.yamato/_packages.yml

```diff
@@ -1719,7 +1719,7 @@ test_core_windows_CUSTOM-REVISION_dependencies:
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
-      -  unity-downloader-cli -u $CUSTOM_REVISION -c editor --wait --published-only
```

Comment:
On other branches it indeed is using the source file argument, but I admit I don't understand why. Aren't we supposed to use the $CUSTOM_REVISION variable?

Reply:
I think this gets passed to the editor-priming job, which uses the $CUSTOM_REVISION and then outputs the unity_revision.txt, and we want to use what the editor priming outputs. 
Although, now I am thinking if this even matters because in both cases the revision should be the same anyways?

Reply:
yeah Liis is correct, the next line down is green and `-  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only`

Reply:
Ah yes, sorry, I forgot how the "old" editor priming works :) 100% makes sense, thanks!